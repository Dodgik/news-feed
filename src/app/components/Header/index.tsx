import * as React from 'react';
import * as moment from 'moment';
import * as style from './style.css';

export class Header extends React.Component {

  render() {
    return (
      <header className={style.header}>
        <h2>Top news of {moment().format('dddd L')}</h2>
      </header>
    );
  }
}
