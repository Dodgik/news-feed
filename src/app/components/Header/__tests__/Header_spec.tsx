// tslint:disable-next-line no-unused-variable
import * as React from 'react';
import * as ReactShallowRenderer from 'react-test-renderer/shallow';

import { Header } from '../../Header';

describe('components/Header', () => {
  it('renders', () => {
    const renderer = new ReactShallowRenderer();
    expect(renderer.render(
      <Header />
    )).toMatchSnapshot();
  });
});
