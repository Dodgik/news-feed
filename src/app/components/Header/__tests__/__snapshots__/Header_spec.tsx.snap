// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Header renders 1`] = `
<header>
  <h2>
    Top news of 
    Monday 05/28/2018
  </h2>
</header>
`;
